package.json

```diff
@@ -1,27 +1,35 @@
 {
-  "author": "Roman Shtylman <shtylman@gmail.com>",
   "name": "localtunnel",
-  "description": "expose localhost to the world",
-  "version": "1.9.2",
+  "description": "Expose localhost to the world",
+  "version": "2.0.0",
   "license": "MIT",
   "repository": {
     "type": "git",
     "url": "git://github.com/localtunnel/localtunnel.git"
   },
+  "author": "Roman Shtylman <shtylman@gmail.com>",
+  "contributors": [
+    "Roman Shtylman <shtylman@gmail.com>",
+    "Gert Hengeveld <gert@hichroma.com> (https://hichroma.com/)",
```

Comment:
Please remove the URL part from the contributors (names and emails only).

Reply:
Done.